test/functional/sendheaders.py

```diff
@@ -4,7 +4,7 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test behavior of headers messages to announce blocks.
 
-Setup: 
+Setup:
 
 - Two nodes, two p2p connections to node0. One p2p connection should only ever
   receive inv's (omitted from testing description below, this is our control).
```

Comment:
nit: Not sure what "omitted from testing description below, this is our control" means. Mind to clarify or remove?

Reply:
I've clarified the language slightly